app/src/main/java/com/andrew/tasky/agenda/presentation/adapters/AttendeeItemAdapter.kt

```diff
@@ -38,17 +38,18 @@ class AttendeeItemAdapter(
     override fun onBindViewHolder(holder: AttendeeItemViewHolder, position: Int) {
         holder.binding.apply {
 
-            val isCreatorHolder = currentList[position].attendeeType == AttendeeType.CREATOR
+            val item = currentList[position]
+
+            val isCreatorHolder = item.attendeeType == AttendeeType.CREATOR
```

Comment:
Here, I'd prefer using a simple boolean for this field (inside the attendee class). The reason is that attendee types might not be mutually exclusive, which an enum class suggests though. So imagine every attendee would also have a certain role or so, maybe `Organizer` or `Presenter`. Your current implementation doesn't allow you to combine these, so that an attendee could be both organizer and creator. 
Right now it works fine as it is because you only use this for visitors and creators, but in case you'd extend this a boolean `isCreator` would make things easier